src/transformers/models/vit/modeling_flax_vit.py

```diff
@@ -418,10 +418,10 @@ def setup(self):
             ),
             dtype=self.dtype,
         )
-        if isinstance(config.pooler_act, str):
-	    self.act = ACT2FN[config.pooler_act]
-	else:
-	    self.act = config.pooler_act
+        if isinstance(self.config.pooler_act, str):
+            self.act = ACT2FN[self.config.pooler_act]
+        else:
+            self.act = self.config.pooler_act
```

Comment:
Activation should be a string, otherwise config will be not serializable

```suggestion
        self.activation_fn = ACT2FN[self.config.pooler_activation_fn]
```

Reply:
oh I see, this is what I did initially but then I saw that one of the activation function related fields mentioned that it is ok to have a function, so I updated it -- will revert